src/seedu/addressbook/ui/TextUi.java

```diff
@@ -93,24 +83,16 @@ public String getUserCommand() {
 
 
     public void showWelcomeMessage(String version, String storageFilePath) {
-        String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
-        showToUser(
-                DIVIDER,
-                DIVIDER,
-                MESSAGE_WELCOME,
-                version,
-                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
-                storageFileInfo,
-                DIVIDER);
+        formatter.createWelcomeMessage(version, storageFilePath, this);
     }
 
     public void showGoodbyeMessage() {
-        showToUser(MESSAGE_GOODBYE, DIVIDER, DIVIDER);
+        formatter.createGoodbyeMessage(this );
```

Comment:
Perhaps there could be a way that doesn't involve bringing the TextUi class into the Formatter.